src/main/resources/templates/index.html

```diff
@@ -14,6 +14,7 @@
 </head>
 <body>
 
+<!-- Having a  bit more of the UI designed would help me visualize how your app would actually work -->
```

Comment:
Nails on a chalkboard.  Here's the specific citation from the assignment:

Note 0: Your GitHub commit should only contain code, not commentary or discussion.  Commentary or discussion belongs in the GitHub pull request, not in source code.   In other words, don't do this: 

// you should really give this a better variable name!

int foo;
 
Instead, change the variable name to a better name:
 
int userAge;